Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -214,7 +215,8 @@ void HGCalGeomParameters::loadGeometryHexagon(const DDFilteredView& _fv,
 					      const std::string & sdTag1,
 					      const DDCompactView* cpv,
 					      const std::string & sdTag2,
-					      const std::string & sdTag3) {
+					      const std::string & sdTag3,
+					      int mode) {
```

Comment:
Hi @bsunanda - could this variable be of the enum type itself rather than an int?